pkg/nimble/rpble/include/nimble_rpble_params.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_nimble_netif_rpble
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for the nimble_netif_rpble module
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NIMBLE_RPBLE_PARAMS_H
+#define NIMBLE_RPBLE_PARAMS_H
+
+#include "nimble_rpble.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Default parameters used for the nimble_netif_rpble module
+ * @{
+ */
+#ifndef NIMBLE_RPBLE_SCAN_ITVL_MS
+#define NIMBLE_RPBLE_SCAN_ITVL_MS       1200U
+#endif
+#ifndef NIMBLE_RPBLE_SCAN_WIN_MS
+#define NIMBLE_RPBLE_SCAN_WIN_MS        120U    /* 10% duty cycling */
```

Comment:
I don't get the connection between 120ms and 10%

Reply:
the Scan interval defaults to 1200ms, while the scan window (the time the radio is actually in RX) is 120ms -> the 120ms time slot is 10% of the scan interval of 1200ms.

Reply:
This information should also be in a comment then.